agent/agent.go

```diff
@@ -285,3 +292,27 @@ func (a *Agent) handlePolicy(p *policy.Policy) {
 		a.policyManager.EnforceCooldown(p.ID, p.Cooldown)
 	}
 }
+
+// enforceActionLimits takes the desired actions and enforces any operator
+// specified limits. The current limits include the policy min and max, as well
+// as the target max change value.
+func (a *Agent) enforceActionLimits(p *policy.Policy, action *strategypkg.Action, status *targetpkg.Status) error {
+
+	if action.Count == strategypkg.MetaValueDryRunCount {
+		return nil
+	}
+
+	// Make sure new count value is within [min, max] limits.
+	action.CapCount(p.Min, p.Max)
+
+	// If the operator has configured a max_change value, attempt to enforce
+	// this.
+	if maxChange, ok := p.Target.Config[targetpkg.ConfigKeyMaxChange]; ok {
+		maxChangeInt, err := strconv.ParseInt(maxChange, 10, 64)
```

Comment:
expanding this parsing to recognize percentages could allow users to specify max changes as a relative value (some percentage of the current value). i think this config could only work when current count is non-zero.

for example, 
```
maxChangeInt = float(maxChangePct) / 100.0 * status.Count
```